src/core/accountAddress.ts

```diff
@@ -496,4 +496,17 @@ Received ${input}`,
     if (this.data.length !== other.data.length) return false;
     return this.data.every((value, index) => value === other.data[index]);
   }
+
+  /**
+   * Returns a lossless short string representation of the address by trimming leading zeros.
+   * If the address consists of all zeros, returns "0".
+   *
+   * @returns A string representation of the address without leading zeros
+   * @group Implementation
+   * @category Serialization
+   */
+  toStringShort(): string {
```

Comment:
This doesn't seem to be including the prefix 0x, should we call it toStringShortWithoutPrefix?